server/templates/oauth_authorized.html

```diff
@@ -0,0 +1,33 @@
+{% extends "fidelity_layout.html" %}
+
+{% block content %}
+
+{% set privacy_policy %}{% include "privacy_policy.html" %}{% endset %}
+
+{% if policy_compare(get_accepted_policy(data['user']), privacy_policy)  %}
+<script>
+    opener.postMessage({type: 'oauth', data: {{ data|tojson }}}, '*');
+    opener.focus();
+    window.close();
+</script>
```

Comment:
small detail, but this part is not indented, while the `{% else %}` part is, would be good to indent here too for consistency.